src/test/java/com/uid2/client/BidstreamClientTests.java

```diff
@@ -33,9 +33,10 @@ public class BidstreamClientTests {
             "UID2, V4",
             "EUID, V4"
     })
-    public void smokeTest(IdentityScope identityScope, TokenVersionForTesting tokenVersion) throws Exception {
-        String advertisingToken = AdvertisingTokenBuilder.builder().withScope(identityScope).withVersion(tokenVersion).build();
-        callAndVerifyRefreshJson(identityScope);
+    public void smokeTestForBidstream(IdentityScope identityScope, TokenVersionForTesting tokenVersion) throws Exception {
+        Instant now = Instant.now();
+        String advertisingToken = AdvertisingTokenBuilder.builder().withScope(identityScope).withVersion(tokenVersion).withEstablished(now.minus(120, ChronoUnit.DAYS)).withGenerated(now.plus(-1, ChronoUnit.DAYS)).withExpiry(now.plus(2, ChronoUnit.DAYS)).build();
```

Comment:
nitpick: better to say `now.minus(1, ChronoUnit.DAYS)` instead of `now.plus(-1, ChronoUnit.DAYS)`